programs/rodeos/streamer_plugin.cpp

```diff
@@ -72,9 +72,13 @@ streamer_plugin::~streamer_plugin() {}
 void streamer_plugin::set_program_options(options_description& cli, options_description& cfg) {
    auto op = cfg.add_options();
    op("stream-rabbits", bpo::value<std::vector<string>>()->composing(),
-      "RabbitMQ Streams to queues if any; Format: amqp://USER:PASSWORD@ADDRESS:PORT/QUEUE[/STREAMING_ROUTE, ...]");
+      "RabbitMQ Streams to queues if any; Additional list of streams delimited by ::: can be specified in the environment variable "
+      EOSIO_STREAM_RABBITS_ENV_VAR
+      ". Format: amqp://USER:PASSWORD@ADDRESS:PORT/QUEUE[/STREAMING_ROUTE, ...].");
    op("stream-rabbits-exchange", bpo::value<std::vector<string>>()->composing(),
-      "RabbitMQ Streams to exchanges if any; Format: amqp://USER:PASSWORD@ADDRESS:PORT/EXCHANGE[::EXCHANGE_TYPE][/STREAMING_ROUTE, ...]");
+      "RabbitMQ Streams to exchanges if any; Additional list of streams delimited by ::: can be specified in the environment variable "
+      EOSIO_STREAM_RABBITS_EXCHANGE_ENV_VAR
+      ". Format: amqp://USER:PASSWORD@ADDRESS:PORT/EXCHANGE[::EXCHANGE_TYPE][/STREAMING_ROUTE, ...]");
```

Comment:
This doesn't make it clear (at least to me) the syntax for multiple. An example might help.

Reply:
Changed to the semantics we discussed. The PR description has a copy of the help text file.